content/events/21-22/04_datacenter2.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Zeus bezoekt het datacenter
+description: Heb je altijd al eens de blinkende lichtjes in het datacenter van dichtbij willen zien? Dit is je kans!
+created_at: 17-09-2021
+time: '19-10-2021 18:00'
+location: Gebouw S10, Campus Sterre, Gent
+locationlink: S10 datacenter
+soon: true
+---
+
+Omdat wij als echte Zeussers en Zeusinnen fan zijn van snelle computers en flikkerende lichtjes organiseren we in samenwerking met de medewerkers van de HPC een rondleiding door het datacenter in gebouw S10. Wil je weten waar Minerva, de Zeus servers en zo veel meer draaien? [Schrijf je dan nu in](https://event.fkgent.be/events/218)!
```

Comment:
Dat is nog een link van 2018

Reply:
Maar in een event dat nog op soon staat is dit nog niet zichtbaar? Moet idd nog geupdate worden maar daar hebben wij de juiste info nu nog niet voor.

Reply:
Ah true